apps/bsd/src/lib/usbstick.ts

```diff
@@ -55,4 +64,5 @@ export const doUnmount = async (): Promise<void> => {
   }
 
   window.kiosk!.unmountUsbDrive(device.deviceName)
+  return new Promise((resolve) => setTimeout(resolve, 2000))
```

Comment:
I would suggest making this longer, say 10s. Until we find a signal that confirms that the data has been written to disk, better safe than sorry, and ejecting disks doesn't happen all that often.